manifests/charts/istio-operator/templates/clusterrole.yaml

```diff
@@ -35,6 +35,18 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - telemetry.istio.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- apiGroups:
+  - extensions.istio.io
+  resources:
+  - '*'
+  verbs:
+  - '*'
```

Comment:
regarding these permissions added, is it possible to make them optional?  I ask because we would not want to use this.

Reply:
@linsun I've removed permissions related to deploy/configmap/webhooks. However I believe it makes sense to include telemetry and extensions permissions in our reader clusterroles. Last time the update to the file was in July  2021, and since the APIs were added after that date, it seems to be outdated - although there was no issue reported since it's not used in the remote cluster service discovery.